recipe/meta.yaml

```
@@ -73,19 +75,38 @@ outputs:
         - tomli-w
         - {{ pin_subpackage('flit-core', exact=True) }}
     test:
+      source_files:
+        - source/tests
+        - source/pyproject.toml
       imports:
         - flit
         - flit.build
       requires:
         - pip
+        - pytest >=2.7.3
+        - testpath
+        - responses
         - ripgrep
       commands:
         - pip check
```

```suggestion
        - echo ON  # [win]
        - pip check
```
Add `echo on` to enable printing command execution on Windows for catching which command fails. By default, it's disabled.